TestAddDelNetworks failure from nc options

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.5.2
```

### Actual behavior

On different distros we may have the 'traditional' or 'openbsd' versions of `nc`.

An unpatched openbsd `nc` will not accept `-l -p 80` - leading to this test failure:


 --- FAIL: TestAddDelNetworks (0.91s)
        network_linux_test.go:599: unexpected failure for "TestHTTPPortmap": failed to open netns "/proc/16444/ns/net": failed to Statfs "/proc/16444/ns/net": no such file or directory
        network_linux_test.go:606: exit status 1
    FAIL
    coverage: 68.5% of statements
    FAIL    github.com/sylabs/singularity/pkg/network       0.939s
    FAIL

--- a/pkg/network/network_linux_test.go
+++ b/pkg/network/network_linux_test.go
@@ -554,7 +554,7 @@ func TestAddDelNetworks(t *testing.T) {
 		{
 			name:    "TestHTTPPortmap",
 			command: "nc",
-			args:    []string{"-l", "-p", "80"},
+			args:    []string{"-l", "80"},
 			runFunc: testHTTPPortmap,
 		},
 		{

